maven-topic1/pom.xml

```diff
@@ -0,0 +1,44 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.bootcamp</groupId>
+	<artifactId>maven-topic1</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	
+	<dependencies>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>log4j</groupId>
+			<artifactId>log4j</artifactId>
+			<version>1.2.17</version>
+		</dependency>
+	</dependencies>
+	
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.1</version>
+				<configuration>
+					<fork>true</fork>
+					<executable>C:\Program Files\Java\jdk1.8.0_131\bin\javac.exe</executable>
```

Comment:
Avoid hard-coding a filesystem path for the executable, this is not portable.

Reply:
It was the only way that i can did it work because when i was trying to compile it tell me that the version of my jdk was not compatible. Before to do that I did all the test as it posible.

Reply:
It's OK, but there is a best way to do that using environmental variable to your JAVA_HOME. Here you have an example https://maven.apache.org/plugins/maven-compiler-plugin/examples/compile-using-different-jdk.html